modules/auxiliary/scanner/mssql/mssql_version.rb

```diff
@@ -0,0 +1,26 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::MSSQL
+  include Msf::Auxiliary::Scanner
+
+  def initialize
+    super(
+      'Name' => 'MSSQL Version Utility',
+      'Description' => 'This module simply queries the MSSQL instance for information.',
+      'Author' => 'MC',
+      'License' => MSF_LICENSE
+    )
+  end
+
+  def run_host(ip)
```

Comment:
Just curious if this module should also support the new MSSQL session type, or if there is a reason we decided not to here? 

I believe both `modules/auxiliary/scanner/mysql/mysql_version.rb` and `modules/auxiliary/scanner/postgres/postgres_version.rb` support their respective session types.

Reply:
Nope, I can't think of a reason, good call out!